example/main.js

```diff
@@ -247,8 +248,7 @@ class App extends React.Component {
   }
 
   handleAddition(tag) {
-    const { tags } = this.state;
-    this.setState({ tags: [...tags, ...[tag]] });
+    this.setState(state => ({ tags: [...state.tags, tag] }));
```

Comment:
can you also pass the delimiters in this file to `Tags` component ?
you can make a constants.js file in the same directory and 
```js
 export const KEYCODES = {
   ENTER: 13,
   COMMA: 188,
}
```
and pass these delimiters.